srcpkgs/hunspell-ro_RO/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'hunspell-ro_RO'
+pkgname=hunspell-ro_RO
+version=3.3.10
+revision=1
+wrksrc=hunspell-ro.${version}
+short_desc="Romanian ro_RO dictionary for hunspell"
+maintainer="Klorophatu <klorophatu@krutt.org>"
+license="GPL-2.0-or-later, LGPL-2.1-or-later, MPL-1.1"
+homepage="https://sourceforge.net/projects/rospell/"
+distfiles="https://sourceforge.net/projects/rospell/files/Romanian%20dictionaries/dict-${version}/hunspell-ro.${version}.tar.gz"
```

Comment:
We have `SOURCEFORGE_SITE` as a variable you can use for downloading stuff from sourceforge, do you think it's possible to plug it in here?